spec/features/posts/post_index.html.erb.spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
```

Comment:
- [ ] Great job so far 👏🏻 but I'm still running fail tests, please, as requested by the previous reviewer, be sure to fix this

![image](https://user-images.githubusercontent.com/79058364/224352927-db6b8b9f-6977-4ea1-8249-334946bd5b95.png)